package.json

```diff
@@ -65,7 +65,9 @@
 	},
 	"devDependencies": {
 		"@babel/core": "7.4.4",
+		"@babel/plugin-proposal-async-generator-functions": "7.2.0",
```

Comment:
This dependency should not be needed as it's already a dependency of `@wordpress/babel-preset-default`.
Same for `@babel/runtime`

Reply:
Nice! I removed those and everything keeps working 👍